observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,107 @@ The Network Observability Operator enables administrators to observe and analyze
+// 
+
+[id="network-observability-network-policy-1-7_{context}"]
+==== Network policy deploys in the FlowCollector custom resource (CR)
+With this release, you can configure the `FlowCollector` CR to deploy a Network policy for Network Observability. Previously, if you wanted a network policy, you had to manually create one. The option to manually create a network policy is still available. For more information, see 
+
+// link to ../../observability/network_observability/network-observability-network-policy[Configuring an ingress network policy by using the FlowCollector custom resource]
+
+[id="network-observability-fips-compliance-1-7_{context}"]
+==== FIPS compliance
+* You can install and use the Network Observability Operator in an {product-title} cluster running in FIPS mode.
++
+--
+include::snippets/fips-snippet.adoc[]
+--
+
+[id="network-observability-dns-enhancements-1-7_{context}"]
+==== eBPF agent enhancements
+The following enhancements are available for the eBPF agent:
+
+* If the DNS service maps to a different port than `53`, you can specify this DNS tracking port.
+* You can now use two ports for transport protocols (TCP, UDP, or SCTP) filtering rules.
+* Filtering on transport ports can now work with a wildcard protocol by leaving the protocol field empty.
+
+[id="network-observability-operator-1-7-bug-fixes_{context}"]
+=== Bug fixes
+
+* Previously, when using a Rhel9.2 real-time kernel, some of the webhooks did not work. Now, a fix is in place to check if this Rhel 9.2 rt kernel is being used, and if it is, produce a warning for the features that do not work, such as packet drop and neither Round-trip Time when using `s390x` architecture. The fix is in OpenShift 4.16+. (link:https://issues.redhat.com/browse/NETOBSERV-1808[*NETOBSERV-1808*])
+* Previously, in the *Manage panels* dialog in the *Overview* tab, filtering on *total*, *bar*, *donut*, or *line* does not show any result. Now the available panels are correctly filtered. (link:https://issues.redhat.com/browse/NETOBSERV-1540[*NETOBSERV-1540*])
+* Previously under high stress, the eBPF agents were susceptible to enter into a state where they generated a high number of small flows, almost not aggregated. With this fix, the aggregation process is still maintained under high stress, resulting in less flows being created. This fix improves the resource consumption not only in the eBPF agent but also in `flowlogs-pipeline` and Loki. (link:https://issues.redhat.com/browse/NETOBSERV-1564[*NETOBSERV-1564*])
+* Previously, when enabling the metric `workload_flows_total` instead of `namespace_flows_total`, the health dashboard stopped showing `By namespace` flow charts. With this fix, the health dashboard now shows these charts by querying the `workload_flows_total` metric. (link:https://issues.redhat.com/browse/NETOBSERV-1746[*NETOBSERV-1746*])
```

Comment:
```suggestion
* Previously, when the `workload_flows_total` metric was enabled instead of the `namespace_flows_total` metric, the health dashboard stopped showing `By namespace` flow charts. With this fix, the health dashboard now shows the flow charts when the `workload_flows_total` is enabled. (link:https://issues.redhat.com/browse/NETOBSERV-1746[*NETOBSERV-1746*])
```